homeassistant/components/sensibo/strings.json

```diff
@@ -67,6 +69,38 @@
           "off": "Off"
         }
       }
+    },
+    "climate": {
+      "climate_device": {
+        "state_attributes": {
+          "fan_mode": {
```

Comment:
Pretty sure these are already translated for climate entities. Why do we need to re-define them ?

Reply:
Not all are translated.